spec/features/courses/description_spec.rb

```diff
@@ -0,0 +1,115 @@
+require 'rails_helper'
+
+feature 'Course description', type: :feature do
+  let(:provider) { jsonapi(:provider, accredited_body?: false) }
+  let(:course) {
+    jsonapi :course,
+      funding: 'fee',
+      site_statuses: [site_status],
+      provider: provider,
+      accrediting_provider: provider
+  }
+  let(:site) { jsonapi(:site) }
+  let(:site_status) do
+    jsonapi(:site_status, :full_time_and_part_time, site: site)
+  end
+  let(:course_response) { course.render }
```

Comment:
Should have a line between `let` and `before` I think. We really need too start listing our specs :(